### ISSUE TITLE
RSS is not updating anymore on Outlook restart

### ISSUE THREAD

**Issue by t347s** [nextcloud/activity#194]
Steps to reproduce

    Activity app > Settings > Enable RSS
    Copy RSS generated link into Outlook
    After is added the RSS flux is working until Outlook is restarted, on reopen the flux is not updated anymore and Outlook gives an error
    Self-signed certificate
    LDAP/AD integration

Expected behaviour

The RSS should work everytime for a fast user files and folder changes knowledge
Actual behaviour

If Outlook is restarted is not updating anymore, the errors are:
Task 'RSS Feeds' reported error (0x800C0019) : 'Synchronization to RSS Feed:"https://subdomain.domain.ext/apps/activity/rss.php?token=oC9Gnj4T5PdM8PcKwLKvAcUSW3hotl" has failed. Outlook cannot download the RSS content from https://subdomain.domain.ext/apps/activity/rss.php?token=oC9Gnj4T5PdM8PcKwLKvAcUSW3hotl because of a problem connecting to the server.'

Task 'RSS Feeds' reported error (0x800C0019) : 'Unknown Error 0x800C0019'
What i tried

    accept certificate - YES - no result
    install certificate - YES - no result
    try multiple browsers/rss agregators - Firefox plugins, Chrome plugins - no result
    cron is running - YES

General server configuration

Operating system: Linux nextcloud 4.4.0-78-generic #99-Ubuntu SMP Thu Apr 27 15:29:09 UTC 2017 x86_64

Web server: Apache/2.4.18 (Ubuntu) (apache2handler)

Database: mysql 5.7.19

PHP version: 7.0.22-0ubuntu0.16.04.1
PHP-modules loaded
Nextcloud configuration

Nextcloud version: 12.0.2 - 12.0.2.0

Updated from an older Nextcloud/ownCloud or fresh install: Updated from 12.0.0 to 12.0.1 to 12.0.2

Are you using external storage, if yes which one: files_external is disabled

Are you using encryption: no
Enabled apps
Disabled apps
Client configuration

Browser: Mozilla/5.0 (Windows NT 6.1; rv:55.0) Gecko/20100101 Firefox/55.0

Operating system: Windows

**Comment by t347s** [comment]
Hi!
Thanks for trying to help me. I allready read for possible fix of this issue on the internet before submit my problem here and i tried some of them with not luck. There is no proxy here. There is no luck with any suggested fix.

**Comment by j-ed** [comment]
If it is related to the CRL access problem, I would start to check which CRL URI is used in your certificate, like
```
# openssl x509 -in your.certificate.com -noout -text | sed -n '/X509v3 CRL Distribution Points:/,$p'
            X509v3 CRL Distribution Points:

                Full Name:
                  URI:http://crl.comodoca.com/COMODORSADomainValidationSecureServerCA.crl
```
Next I would try to untick the parameter `Internet Options->Advanced->Check for server certificate revocation` to prevent the system from checking CRLs.

**Comment by t347s** [comment]
Hi!
Thanks j-ed for your advices, unfortunately none of this is the solution. I just updated NC to 12.0.3 and just no. I tried with another software to read Activity RSS (in my case RSSOwl) and it's a miracle, is working, every time i open this software the feed's are downloaded. But this is not o solution for me, i am interested in Outlook to work because this program is "live" most of the time and i like the to have one software running, not multiple programs for every task.

Best regards
